src/monitoring_service/database.py

```diff
@@ -102,23 +106,29 @@ def upsert_channel(self, channel: Channel) -> None:
         )
         self.conn.execute(upsert_sql, values)
 
-    def get_channel(self, token_network_address: str, channel_id: int) -> Optional[Channel]:
+    def get_channel(
+        self, token_network_address: TokenNetworkAddress, channel_id: int
+    ) -> Optional[Channel]:
         row = self.conn.execute(
             """
                 SELECT * FROM channel
                 WHERE identifier = ? AND token_network_address = ?
             """,
-            [hex(channel_id), token_network_address],
+            [hex(channel_id), to_checksum_address(token_network_address)],
         ).fetchone()
 
         if row is None:
             return None
         kwargs = {
             key: val for key, val in zip(row.keys(), row) if not key.startswith("update_status")
         }
+        kwargs["token_network_address"] = decode_hex(kwargs["token_network_address"])
```

Comment:
Mh, I'd perfer to use `to_canonical_address` for hex to bytes conversion. I think it makes the intention clearer.

Reply:
But that returns eth_typing.Address, which we are not using, so I expect that we'll get type errors. Do you agree?

And why don't we use eth_typing.Address as address type?

Reply:
In a quick check mypy didn't complain, which is a bit surprising.

And I think the `eth_typing` package is quite new, so we might want to use the Raiden types as aliases.

Reply:
Anyways, we can fix the `decode_hex` stuff later.